code/modules/battlemonsters/datum_elements.dm

```diff
@@ -146,41 +149,63 @@
+	name = "Dragon God"
+	id = "dragon_god"
+	icon_state = "dragon_god"
+	defense_type = BATTLE_MONSTERS_DEFENSETYPE_GOD | BATTLE_MONSTERS_DEFENSETYPE_GIANT_DRAGON
+	elements = BATTLE_MONSTERS_ELEMENT_ENERGY
+	description = "Their %ELEMENT_AND wings are absolutely massive, and their breath constantly exhales deadly %ELEMENT_AND. This being is a common god among dragons, and it is theorized that one of the many breeds of dragons came from %NAME."
+	special_effects = "Godly Protection: %ELEMENT_OR monsters cannot attack or be attacked by %NAME.<br>Wingspan: When %NAME is summoned, all non-flying monsters on the field are sent to the graveyard."
+	tip = "%NAME can still be affected by traps or spells.<br>%NAME can also send friendly units to the graveyard."
+	power_add = 2000
+	rarity = BATTLE_MONSTERS_RARITY_LEGENDARY
+	rarity_score = 3
+
 /datum/battle_monsters/element/water
 	name = "Water"
 	id = "water"
 	icon_state = "water"
-	description = "Their %WEAPON_AND is coated with magical water."
+	description = "Their %WEAPON_AND is coated with magical shifting water."
 	elements = BATTLE_MONSTERS_ELEMENT_WATER
 	rarity = BATTLE_MONSTERS_RARITY_COMMON
 
 /datum/battle_monsters/element/water/morphing
 	name = "Morphling"
 	id = "morphling"
 	icon_state = "water2"
-	description = "Upon closer examination, it appears that they can morph into various shapes using the magic of %ELEMENT_AND."
+	description = "Upon closer examination, it appears that they can morph into various shapes using the magic of %ELEMENT_AND. Morphlings can common be seen near rivers, lakes, or streams harrassing travelers who dare to take a drink from the protected waters."
 	special_effects = "Morph: %NAME is automatically switched to defense mode if they attack, and automatically switched to attack mode if they succesfully defend from an attack. %NAME cannot be switched to attack mode or defense mode manually."
 	rarity = BATTLE_MONSTERS_RARITY_UNCOMMON
 
 /datum/battle_monsters/element/water/raincloud
 	name = "Rain"
 	id = "rain"
 	icon_state = "raincloud"
-	description = "Depressing rainclouds hover above them."
+	description = "Depressing rainclouds hover above them. Such rainclouds are usually caused by a curse given by witches for the penalty of being too sad."
 	special_effects = "Dark Rainclouds: %NAME is immune to all non %ELEMENT_OR based trap and spell cards as long as %NAME attacked on the owner's previous turn."
 	power_mul = 0.75
 	attack_add = 0.5
 	rarity = BATTLE_MONSTERS_RARITY_UNCOMMON
 
 /datum/battle_monsters/element/water/trench_pirate
-	name = "Trench Pirate"
+	name = "Trench"
 	id = "trench_pirate"
-	description = "They're covered entirely with seaweed and barnacles."
+	description = "They're covered entirely with seaweed and barnacles. They wear an eyepatch and a bandana, and wield a %WEAPON_AND fearsomely. Rumor has it that Trench beings were shipwrecked sailors whose bodies sunk into marine trenches into the hells below."
```

Comment:
Latter sentence here is a bit awkward. Maybe find a way to phrase it without two intos?